### PR TITLE
Update adding-a-new-ssh-key-to-your-github-account.md

### PR DESCRIPTION
**Pull request by brockneedscoffee**
The updated docs that use ed25519 to generate the key are merged.  This needs to match those steps

### Why:

The updated docs introduce in [this PR](https://github.com/github/docs/pull/362) now make these steps incorrect, this just updates the steps to match the key generation now the guidance no longer uses rsa

### What's being changed:

I updated the copy step to read:

```markdown
pbcopy < ~/.ssh/id_ed25519.pub
```

### Check off the following:
- [X] All of the tests are passing.
- [X] I have reviewed my changes in staging.
- [X] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [X] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
Thanks for catching that and opening a PR @brockneedscoffee! I'll get this triaged for review 🎉

**Comment by brockneedscoffee** [comment]
@janiceilene any updates on this?

File: content/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account.md
Comment (shati-patel) [inline content/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account.md:28]:
```suggestion
  # Copies the contents of the id_ed25519.pub file to your clipboard
```

Could you update this line to match your other change? 😊

**Review by shati-patel — DISMISSED** [review]
Thank you for the pull request, @brockneedscoffee! We appreciate the help 🎉 

There are a few other places in this file where `id_rsa` appears. For consistency, could you also update those? I've added one of them as a suggestion, but there are 4 more further down in the file (in the Windows/Linux sections). 

Once you've done that, we can get this merged! Thank you again for you contribution 😊

File: content/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account.md
Comment (felicitymay) [reply content/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account.md:28]: Hi @brockneedscoffee - there seem to be several other problems with this topic, so I'm going to commit the reviewer's suggestion and merge your PR to unblock fixes to other areas of the topic.

**Review by felicitymay — APPROVED** [review]
I agree with the reviewer that it would be better to update all the instances of `id_rsa` in this topic, but there are other problems with this topic that need to be fixed to make it usable. So I'm going to merge this as-is.

**Pull request merged**
